base/applications/mscutils/eventvwr/lang/zh-CN.rc

```diff
@@ -85,7 +85,7 @@ FONT 9, "宋体", 400, 0, 0x1
 BEGIN
     LTEXT "日期：", IDC_STATIC, 8, 5, 31, 8
     EDITTEXT IDC_EVENTDATESTATIC, 46, 5, 47, 8, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
-    LTEXT "来源：", IDC_STATIC, 104, 5, 36, 8
+    LTEXT "来源：", IDC_STATIC, 103, 5, 36, 8
```

Comment:
Is this change to make it compatible with the en-us translation?

Reply:
The 103 is both: a sync to en-US, but it will other horizontally-align with other controls below that do also use the 103:, e.g. line 92
`    LTEXT "分类：", IDC_STATIC, 103, 15, 36, 8`
and line 96
`LTEXT "事件 ID：", IDC_STATIC, 103, 25, 36, 8`
and so on and so forth.